perfmetrics/scripts/hns_rename_folders_metrics/renaming_benchmark.py

```diff
@@ -45,7 +51,7 @@
 log = logging.getLogger()
 
 
-def _upload_to_gsheet(worksheet, data, spreadsheet_id) -> (int):
+def _upload_to_gsheet(worksheet, data,vm_worksheet,vm_data, spreadsheet_id) -> (int):
```

Comment:
Keep single argument for worksheet and data.

Reply:
Fixed.